spec/02-integration/04-admin_api/02-apis_routes_spec.lua

```diff
@@ -206,7 +260,7 @@ describe("Admin API #" .. kong_config.database, function()
         --
         -- Eventually, our Admin endpoint will follow a more appropriate
         -- behavior for PUT.
-        local res = assert(client:send {
+        local res = assert(helpers.admin_client():send {
```

Comment:
Curious: why not using `client` here?

Reply:
Because the test generates 404 and it makes the client really flaky as you see it runs this `it_content_types` loop, so first time it works, but subsequently you start getting:

> spec/02-integration/04-admin_api/02-apis_routes_spec.lua:263: closed

The client is really problematic, because you cannot really call `:send` multiple times without getting into issues. That's why. To make tests pass reliably.

Reply:
Ohh I see - hmmm